test-files/cmd/slow/NodeDensityTilesCmdTest/output-cropped.geojson

```diff
@@ -1,10 +1,6 @@
-{
-"type": "FeatureCollection",
-"crs": { "type": "name", "properties": { "name": "urn:ogc:def:crs:OGC:1.3:CRS84" } },
-"features": [
-{ "type": "Feature", "properties": { "node_count": "659", "task": "1" }, "geometry": { "type": "MultiPolygon", "coordinates": [ [ [ [ -77.0485478, 38.885505 ], [ -77.0485478, 38.895505 ], [ -77.0395478, 38.895505 ], [ -77.0395478, 38.885505 ], [ -77.0485478, 38.885505 ] ] ] ] } },
-{ "type": "Feature", "properties": { "node_count": "410", "task": "2" }, "geometry": { "type": "MultiPolygon", "coordinates": [ [ [ [ -77.0485478, 38.895505 ], [ -77.0485478, 38.900505 ], [ -77.0395478, 38.900505 ], [ -77.0395478, 38.895505 ], [ -77.0485478, 38.895505 ] ] ] ] } },
-{ "type": "Feature", "properties": { "node_count": "611", "task": "3" }, "geometry": { "type": "MultiPolygon", "coordinates": [ [ [ [ -77.0395478, 38.885505 ], [ -77.0395478, 38.891505 ], [ -77.0275478, 38.891505 ], [ -77.0275478, 38.885505 ], [ -77.0395478, 38.885505 ] ] ] ] } },
-{ "type": "Feature", "properties": { "node_count": "730", "task": "4" }, "geometry": { "type": "MultiPolygon", "coordinates": [ [ [ [ -77.0395478, 38.891505 ], [ -77.0395478, 38.900505 ], [ -77.0275478, 38.900505 ], [ -77.0275478, 38.891505 ], [ -77.0395478, 38.891505 ] ] ] ] } }
-]
+{"generator":"Hootenanny","type":"FeatureCollection","bbox": [-77.049, 38.886, -77.028, 38.901],"import_url":"hootapidb://hoot@localhost:5432/hoot/DcTigerRoads;osmapidb://hoot@localhost:5432/osmapi_test","features": [
+{"type":"Feature","properties":{"task":"1","node_count":"659"},"geometry": {"type":"MultiPolygon","coordinates": [[[[-77.0485478, 38.885505],[-77.0485478, 38.895505],[-77.0395478, 38.895505],[-77.0395478, 38.885505],[-77.0485478, 38.885505]]]]}},
+{"type":"Feature","properties":{"task":"2","node_count":"410"},"geometry": {"type":"MultiPolygon","coordinates": [[[[-77.0485478, 38.895505],[-77.0485478, 38.900505],[-77.0395478, 38.900505],[-77.0395478, 38.895505],[-77.0485478, 38.895505]]]]}},
+{"type":"Feature","properties":{"task":"3","node_count":"611"},"geometry": {"type":"MultiPolygon","coordinates": [[[[-77.0395478, 38.885505],[-77.0395478, 38.891505],[-77.0275478, 38.891505],[-77.0275478, 38.885505],[-77.0395478, 38.885505]]]]}},
+{"type":"Feature","properties":{"task":"4","node_count":"730"},"geometry": {"type":"MultiPolygon","coordinates": [[[[-77.0395478, 38.891505],[-77.0395478, 38.900505],[-77.0275478, 38.900505],[-77.0275478, 38.891505],[-77.0395478, 38.891505]]]]}}]
```

Comment:
This is the new output from `OsmGeoJsonWriter` that replaces `ogr2ogr`.  The differences are trivial, `generator` is ignored, `crs` defaults to `CRS84` anyway, we've added the `import_url` and the order of the `properties` is different.

Reply:
@brianhatchl here is the new output for the tiles with the `import_url` included.  Does this look right to you?

Reply:
somehow missed this mention notification.  thought the property key was going to be `source`.  will it be 'import_url` for files too?

Reply:
I honestly kinda liked the abbreviated value too `hootapidb:3431`

Reply:
and probably just the filename, not the path.  Though it seems the command stores whatever was passed in, so it could be absolute or relative path or none at all.

Reply:
@brianhatchl I actually started with `source` but while looking into Tasking Manager they have a property `import_url` that looked like it might be already something you could use without having to modify anything (or at least not much).  I came across it here:

https://wiki.openstreetmap.org/wiki/Tasking_manager_admin

As for the filename vs pathname, I'm open to either.  I can also use `hootapidb:<map id>` if that is what you prefer.  I just thought the full URL might give you more information but if that isn't needed I can make the change.

Reply:
Ok, let's switch it back to `source` as the property name and have it live at root level.  The value for hoot db layers will be `hootapidb:map_id` and for files it will just be the filename with no path.

The `import_url` is interesting.  Seems to support an integration with JOSM:

>            ${_('Put here anything that can be useful to users while taking a task. If you have added extra properties within the GeoJSON of the task, they can be referenced by surrounding them in curly braces. For eg. if you have a property called "import_url" in your GeoJSON, you can reference it like:')} « <em>${_('This task involves loading extra data. Click [here](http://localhost:8111/import?new_layer=true&url={import_url}) to load the data into JOSM')}</em> ».

This might be more useful when we attempt to split up the secondary data and have it clipped per task.  Then a property on each task feature will be necessary, though we'll build our own integration with hoot UI for that.  One day if we have a JOSM Hoot plugin we can properly leverage the `import_url`.

Reply:
Ok sounds good.  I'll make the changes.